### DISCUSSION TITLE
How do i install models ?

### DISCUSSION THREAD

**Discussion by dakeeper** [matiasdelellis/facerecognition#696] [Q&A]
Hello, 
i want to test face recognition app, dlib is installed with no problems.
But how do i install the Model(s) ?
do i need to download them to any special folder or what?

i tried 
face:setup --model 1
using the occ web. But nothing happens.

Can anybody help me please?

**Comment by matiasdelellis** [comment]
I have never tried occ web, is it impossible for you to run it natively?
